test-harness/index.js

```diff
@@ -0,0 +1,270 @@
+const fetch = require('node-fetch');
+const requests = require('./requests');
+
+const { exec } = require('child_process');
+const { makeRequest, constructMasterFileName, readFile } = require('./helpers');
+
+async function waitForPrism(done) {
+  try {
+    await fetch('http://localhost:4010');
+    done();
+  } catch (err) {
+    setTimeout(() => {
+      waitForPrism(done);
+    }, 500);
+  }
+}
+
+async function runTest(req) {
+  const { dynamic, ...request } = req;
+
+  const masterFileName = constructMasterFileName(request);
+  const masterFile = readFile(masterFileName);
+
+  const reqRes = await makeRequest(request);
+
+  return {
+    reqRes,
+    masterFile,
+  };
+}
+
+function killPrism(done = () => null) {
+  exec('fuser -k 4010/tcp', () => {
+    done();
+  });
+}
+
+const spec = `${process.env.SPEC || `${__dirname}/../examples/petstore.oas2.json`}`;
```

Comment:
Use `path.join` so you can avoid the string interpolation.
```suggestion
const spec = process.env.SPEC || path.join(__dirname, '/../examples/petstore.oas2.json');
```

Reply:
@XVincentX, I don't see any merit it using `path.join` instead of string interpolation. How exactly is `path.join` better here?

Reply:
There are supposedly some windows support benefits or something. This is the way it's generally done everywhere else so we might as well for consistency even if nothing else.

Reply:
Yeah exactly, in general `path.join` is going to be always better than any implementation we can come up with. It's clearly not blocking, just saying.

Reply:
@lag-of-death this was the suggestion i was suggesting you implemented.